examples/kustomization/base/kustomization.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: minio-tenant
+resources:
+  - namespace.yaml
+  - tenant-config.yaml
+  - tenant-minio-config_deprecated.yaml
```

```suggestion
  - tenant-minio-config_deprecated.yaml
```

why is this deprecate.yaml?

---

we are passing the tenant credentials via the `tenant-config.yaml`, configuration values there will have more precedence than `credsSecret` or tenant env variables, also credentials will not appear if someone do `docker inspect` of a minio pod